src/pages/settings/Members.tsx

```diff
@@ -0,0 +1,284 @@
+
+gql`
+  query getInvites($page: Int, $limit: Int) {
+    invites(page: $page, limit: $limit) {
+      metadata {
+        currentPage
+        totalPages
+        totalCount
+      }
+      collection {
+        ...InviteItem
+      }
+    }
+  }
+
+  query getMembers($page: Int, $limit: Int) {
+    memberships(page: $page, limit: $limit) {
+      metadata {
+        currentPage
+        totalPages
+      }
+      collection {
+        ...MembershipItem
+      }
+    }
+  }
+
+  ${InviteItemFragmentDoc}
+  ${MembershipItemFragmentDoc}
+`
+
+const Members = () => {
+  const { translate } = useInternationalization()
+  const revokeInviteDialogRef = useRef<RevokeInviteDialogRef>(null)
+  const revokeMembershipDialogRef = useRef<RevokeMembershipDialogRef>(null)
+  const createInviteDialogRef = useRef<CreateInviteDialogRef>(null)
+  const {
+    data: invitesData,
+    error: invitesError,
+    loading: invitesLoading,
+    refetch: invitesRefetch,
+    fetchMore: invitesFetchMore,
+  } = useGetInvitesQuery()
+  const {
+    data: membersData,
+    error: membersError,
+    loading: membersLoading,
+    refetch: membersRefetch,
+    fetchMore: membersFetchMore,
+  } = useGetMembersQuery()
+  const hasInvites = !!invitesData?.invites.metadata.totalCount
+  const invitesMetadata = invitesData?.invites.metadata
+  const membersMetadata = membersData?.memberships.metadata
+
+  return (
+    <Page>
+      <Title variant="headline">{translate('text_63208b630aaf8df6bbfb2657')}</Title>
+      <Subtitle>{translate('text_63208b630aaf8df6bbfb2659')}</Subtitle>
+      {(!!invitesLoading || !!invitesError || !!hasInvites) && (
+        <Head>
+          {!!invitesLoading ? (
+            <TitleSkeleton variant="text" height={12} width={160} />
+          ) : (
+            <>
+              <Typography variant="subhead">
+                {translate('text_63208b630aaf8df6bbfb265d')}
+              </Typography>
+              {!!hasInvites && (
+                <Button
+                  variant="secondary"
+                  onClick={() => {
+                    createInviteDialogRef.current?.openDialog()
+                  }}
+                >
+                  {translate('text_63208b630aaf8df6bbfb265b')}
+                </Button>
+              )}
+            </>
+          )}
+        </Head>
+      )}
+      {!!invitesLoading ? (
+        <LoadingListWrapper>
+          {[1, 2].map((i) => (
+            <InviteItemSkeleton key={i} />
+          ))}
+        </LoadingListWrapper>
+      ) : (
+        <InvitationsListWrapper>
+          {!invitesLoading && !!invitesError ? (
+            <ErrorPlaceholder
+              noMargins
+              title={translate('text_6321a076b94bd1b32494e9e6')}
+              subtitle={translate('text_6321a076b94bd1b32494e9e8')}
+              buttonTitle={translate('text_6321a076b94bd1b32494e9ea')}
+              buttonVariant="primary"
+              buttonAction={() => invitesRefetch()}
```

Comment:
```
buttonAction={invitesRefetch}
```

no need to recreate a wrapper function here

Reply:
ok nice ! Updated